shuup/admin/modules/orders/views/addresses.py

```diff
@@ -26,6 +26,10 @@ class OrderAddressEditView(UpdateView):
     context_object_name = "order"
     template_name = "shuup/admin/orders/_address_edit.jinja"
 
+    def get_queryset(self):
+        shop_ids = Shop.objects.get_for_user(self.request.user).values_list("id", flat=True)
+        return Order.objects.exclude(deleted=True).filter(shop_id__in=shop_ids)
```

Comment:
why not just
```py
Order.objects.exclude(deleted=True).filter(shop_in=Shop.objects.get_for_user(get_shop(self.request)))
```

Reply:
get_for_user does not take shop as param :)